lib/wrap-request.js

```diff
@@ -0,0 +1,38 @@
+module.exports = wrapRequest
+
+const noop = () => Promise.resolve()
+
+function wrapRequest (octokit, emitter, globalLimiter, writeLimiter, request, options) {
+  return globalLimiter.schedule(async function () {
+    if (options.method !== 'GET') {
```

Comment:
Also check for `HEAD`: https://developer.github.com/v3/#http-verbs

Reply:
Nice catch! 👍 Fixing it.